app/scripts/modules/core/src/pipeline/config/triggers/artifacts/helm/HelmArtifactEditor.tsx

```diff
@@ -0,0 +1,94 @@
+import { module } from 'angular';
+import { react2angular } from 'react2angular';
+import { ArtifactService } from 'core/pipeline/config/triggers/artifacts/ArtifactService';
+import { Option } from 'react-select';
+import { isFinite } from 'lodash';
+
+interface IState {
+  names: string[];
+  versions: string[];
+  artifact: IArtifact;
+}
+
+export class HelmArtifactEditor extends React.Component<IArtifactEditorProps, IState> {
+  constructor(props: IArtifactEditorProps) {
+    super(props);
+    this.state = { names: [], versions: [], artifact: this.props.artifact };
+    this.state.artifact.artifactAccount = this.props.account.name;
+    ArtifactService.getArtifactNames('helm', this.props.account.name).then(names => {
+      this.setState({ names });
+    });
+  }
+
+  public componentWillReceiveProps(nextProps: IArtifactEditorProps) {
+    if (this.props.account.name !== nextProps.account.name) {
+      this.state.artifact.artifactAccount = nextProps.account.name;
+      this.state.artifact.name = '';
+      this.state.artifact.version = '';
+      nextProps.onChange({ ...this.state.artifact });
+      ArtifactService.getArtifactNames('helm', nextProps.account.name).then(names => {
+        this.setState({ names, versions: [] });
+      });
+    }
+  }
+
+  public render() {
+    const nameOptions = this.state.names.map(name => ({ value: name, label: name }));
+    const versionOptions = this.state.versions.map(version => ({ value: version, label: version }));
+    const labelColumns = isFinite(this.props.labelColumns) ? this.props.labelColumns : 2;
+    const labelClassName = 'col-md-' + labelColumns;
+    return (
+      <div className="form-group row">
+        <label className={labelClassName + ' sm-label-right'}>Name</label>
+        <TetheredSelect
+          className={'col-md-3'}
+          options={nameOptions}
+          value={this.state.artifact.name || ''}
+          onChange={(e: Option) => {
+            this.onChange(e, 'name');
+            this.onNameChange();
+          }}
+          clearable={false}
+        />
+        <label className={'col-md-2 sm-label-right'}>Version</label>
+        <TetheredSelect
+          className={'col-md-3'}
+          options={versionOptions}
+          value={this.state.artifact.version || ''}
+          onChange={(e: Option) => {
+            this.onChange(e, 'version');
+          }}
+          clearable={false}
+        />
+      </div>
+    );
+  }
+
+  private onChange = (e: Option, field: keyof IArtifact) => {
+    this.state.artifact[field] = e.value.toString();
+    this.props.onChange({ ...this.state.artifact });
+  };
+
+  private onNameChange = () => {
+    ArtifactService.getArtifactVersions('helm', this.state.artifact.artifactAccount, this.state.artifact.name).then(
+      versions => {
+        this.setState({ versions });
+      },
+    );
+  };
+}
+
+export const ARTIFACT_ACCOUNT_SELECTOR_COMPONENT_REACT = 'spinnaker.core.artifacts.account.selector.react';
```

Comment:
It looks like this is copy/paste from the ArtifactAccountSelector.tsx but not used anywhere else in this PR.  I think you can safely delete lines 83 - 94.

If you do need this module declaration for some reason then change these lines to something like 
```js
export const HELM_ARTIFACT_SELECTOR_COMPONENT_REACT = 'spinnaker.core.artifacts.helm.selector.react';
module(HELM_ARTIFACT_SELECTOR_COMPONENT_REACT, []).comp// ... and so on
```

Reply:
It's my fault. I deleted these lines